src/app/app.server.ts

```diff
@@ -0,0 +1,40 @@
+import express, { Express } from 'express';
+import morgan from 'morgan';
+import config from '../config';
+import cakeRoutes from './cakes';
+
+const app: Express = express();
+
+// Config middleware
+app.use(morgan('dev'));
+
+// config routes
+app.use('/api/cakes', cakeRoutes);
```

Comment:
Remember, the routes should be running under the root URI: `/cakes`. The `api` prefix should not be included.

Reply:
Oops, my bad; I'll update the route later today, thanks!